app/assets/images/payment_icons/bc_card.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" fill="none" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-bc-card"><title id="pi-bc-card">BC Card</title><rect width="38" height="24" rx="2" fill="#fff"/><rect x=".5" y=".5" width="37" height="23" rx="1.5" stroke="#000" stroke-opacity=".07"/><path d="M28 12.001A9 9 0 0119 21c-4.972 0-9-4.03-9-8.999C10 7.031 14.028 3 19 3c4.97 0 9 4.031 9 9.001" fill="#E83D44"/><path d="M11.498 10.997l.015.2a.11.11 0 00.063.103c.036.016.078.008.112-.023.337-.281.846-.533 1.255-.391.318.11.537.44.65.983.233 1.133-.547 1.68-1.031 1.905l-.018.007-.622.307s-.035.013-.072-.006c-.038-.019-.052-.044-.075-.11-.424-1.563-.567-3.203-.41-4.73a.77.77 0 01.16-.371l.517-.556.06-.064c.287-.277.55-.413.693-.354.123.051.152.242.155.393.022.813-.431 1.644-1.389 2.54 0 0-.07.06-.063.167zm2.867-1.127a1.397 1.397 0 00-.93-.172c.592-.627.912-1.311.912-2 0-.063-.002-.127-.008-.19-.033-.43-.233-.75-.534-.86-.31-.113-.87-.057-1.767.856l-.535.602c-.146.181-.71 1.034-.817 1.922-.173 1.511-.062 3.018.323 4.356.045.155.186.534.535.674.37.149.718-.03 1.027-.191l.081-.04.32-.143.31-.137.378-.165c.27-.11.525-.214.77-.422.332-.28.573-.65.698-1.067a3.67 3.67 0 00.132-.721c.019-.21.038-.612-.038-.968-.17-.783-.56-1.157-.857-1.334zM21.429 5.127c-.175-.599-.463-.726-.545-.754a4.737 4.737 0 00-2.276-.155c-1.133.208-2.116.802-2.763 1.674-.682.916-.944 2.056-.742 3.212.204 1.16.792 2.1 1.7 2.715 1.066.72 2.486.938 4.001.617.895-.192 1.638-.542 2.212-1.037a.903.903 0 00.307-.704s.01-.16-.113-.663c-.033-.124-.086-.159-.172-.159-.063 0-.13.071-.143.086-.426.495-1.08.839-1.895 1-2.085.401-3.692-.574-4.09-2.486-.17-.827-.028-1.622.401-2.239.435-.623 1.147-1.036 2.003-1.166a3.624 3.624 0 011.916.236s.254.073.199-.177" fill="#fff"/></svg>
```

Comment:
There are some border radius differences: [video demonstration](https://screenshot.click/02-23-ix6ie-2ikwz.mp4)

I believe it's a setting change between "inner border" and "outer border" in the design app.